app/presenters/measure_presenter.rb

```diff
@@ -1,28 +1,27 @@
-class MeasurePresenter
-  attr_reader :measure
-
-  def initialize(measure)
-    @measure = measure
+class MeasurePresenter < SimpleDelegator
+  def has_children_geographical_areas?
+    geographical_area.children_geographical_areas.any?
   end
 
-  def geo_class
```

Comment:
The second branch of this method is impossible to hit so I've removed it and updated downstream